pyteal/ast/multi_test.py

```diff
@@ -37,7 +37,7 @@ def __test_single_conditional(
     ifExpr = (
         pt.If(expr.output_slots[1].load())
         .Then(expr.output_slots[0].load())
-        .Else(pt.Bytes("None"))
+        .Else(pt.App.globalGet(pt.Bytes("None")))
```

Comment:
This unit test was previously passing despite its then and else branches returning different types for some iterations.

To fix it, I've made the else branch return `App.globalGet`, which is a `TealType.any`, so it will be correct for all then branches unless the then branch is `TealType.none`.

Reply:
I see. I believe @algoidurovic wrote this test originally, so I'm curious if he has any thoughts about changing this test.